src/front-door/azext_front_door/_params.py

```diff
@@ -164,6 +164,8 @@ def load_arguments(self, _):
         c.argument('redirect_url', help='URL used for redirect rule action.')
         c.argument('custom_block_response_status_code', help='HTTP status to return for blocked requests.')
         c.argument('custom_block_response_body', help='Body to return for blocked requests.')
+        c.argument('request_body_check', arg_type=get_enum_type(PolicyRequestBodyCheck), help='Disabled or Enabled status. Default value is Disabled')
```

Comment:
PolicyRequestBodyCheck only have "Enabled" and "Disabled" values. Please use get_three_state_flag 

``` python
arg_type=get_three_state_flag(positive_label='Enabled', negative_label='Disabled', return_label=True)
```

Reply:
Replaced with get_three_state_flag